src/dos/dos_files.cpp

```diff
@@ -871,7 +872,9 @@ Bit8u FCB_Parsename(Bit16u seg,Bit16u offset,Bit8u parser ,char *string, Bit8u *
 	/* do nothing if not a valid name */
 	if(!isvalid(string[0])) goto savefcb;
 
-	hasname=true;finished=false;fill=' ';index=0;
+	hasname = true;
+	fill = ' ';
+	index = 0;
 	/* Copy the name */	
```

Comment:
Ugh - my eyes can't even look at that original code `hasname=true;finished=false;fill=' ';index=0;` without cramping up! 

Is clang-format capable of fixing such garbage? (I can't fathom the motivation to write code like this - it doesn't even have spaces.. who is benefiting from this?).  :exploding_head:

Reply:
clang-format can split it :) I just pressed Ctrl+K in Vim on line 874 and it got placed in separate lines.

No idea why anyone thought code like this is ok to write in the first place… My eyes are just glancing over it when reading. And seemingly author forgot about `finished` variable as well.